packages/client/src/components/World/index.tsx

```diff
@@ -24,8 +24,19 @@ import GameServerWarnings from './GameServerWarnings'
 import { initEngine, retriveLocationByName, teleportToLocation } from './LocationLoadHelper'
 import { teleportPlayer } from '@xrengine/engine/src/avatar/functions/teleportPlayer'
 import { Network } from '@xrengine/engine/src/networking/classes/Network'
-import { NetworkSchema } from '@xrengine/engine/src/networking/interfaces/NetworkSchema'
+import { SystemUpdateType } from '../../../../engine/src/ecs/functions/SystemUpdateType'
+import { GameManagerSystem } from '../../../../engine/src/game/systems/GameManagerSystem'
```

Comment:
This doesn't exist any more I don't think, or is throwing an error when I try to run this branch.